content/v2.0/reference/api/client-libraries/js.md

```diff
@@ -1,12 +1,167 @@
+2. Navigate to examples directory and install NPM
+    ```sh
+    cd examples 
+    npm install
+    ```
+3. Update your `./env` and `index.html` with the name of your [bucket](/v2.0/organizations/buckets/), [organization](/v2.0/organizations/), [token](/v2.0/security/tokens/), and `url` which relies upon proxy to forward requests to the target InfluxDB. 
+4. Run the following command to run the application at [http://localhost:3001/examples/index.html]()
+```sh
+npm run browser
+```
+
+
+## Write data to InfluxDB with JavaScript
+
+We are going to write some data as a point using the JavaScript library.
+
+In your JavaScript script, import the latest release from the NPM repository. 
+```js
+import {InfluxDB, Point} from 'https://unpkg.com/@influxdata/influxdb-client/dist/index.browser.mjs'
+```
+
+Next, define a few constant variables with the name of your [bucket](/v2.0/organizations/buckets/), [organization](/v2.0/organizations/), [token](/v2.0/security/tokens/), and `url` which  relies upon proxy to forward requests to the target InfluxDB. 
+
+```js
+const url = '/influx' 
+const token = 'my-token'
+const org = 'my-org'
+const bucket = 'my-bucket'
+```
+
+In order to write data, we need to instantiate the InfluxDB JavaScript Client and pass in our named parameters: `url` and `token`.
+
+```js
+const influxDB = new InfluxDB({url, token})
+```
+
+We get a write client with the `getWriteApi` method and pass in our other named parameters: `org` and `bucket`. 
+
+```js
+const writeApi = influxDB.getWriteApi(org, bucket)
+```
+
+We need three more lines for our program to write data.
+The `useDefaultTags` method instructs the write api to use default tags when writing points. Create a [point](/v2.0/reference/glossary/#point) and write it to InfluxDB using the `writePoint` method. The `tag` and `floatField` methods add key value pairs for the tags and fields, respectively.  Close the client to flush all pending writes and finish. 
+
+```js
+writeApi.useDefaultTags({location: 'browser'})
+const point1 = new Point('temperature')
+          .tag('example', 'index.html')
+          .floatField('value', 24)
+        writeApi.writePoint(point1)
+        console.log(`${point1}`)
+writeApi.close()
+```
+
+### Complete example write script
+
+```js
+const writeApi = new InfluxDB({url, token})
+const writeApi = influxDB.getWriteApi(org, bucket)
+// setup default tags for all writes through this API
+writeApi.useDefaultTags({location: 'browser'})
+const point1 = new Point('temperature')
+  .tag('example', 'index.html')
+  .floatField('value', 24)
+writeApi.writePoint(point1)
+console.log(` ${point1}`)
+// flush pending writes and close writeApi
+writeApi
+  .close()
+  .then(() => {
+    console.log('WRITE FINISHED')
+  })
+```
+
+## Query data from InfluxDB with JavaScript
+
+We get a query client with the `getQueryApi` method and pass in our named parameter: `org`. 
```

Comment:
Use the `getQueryApi` method of the `InfluxDB` client to create a new `query client`. Provide your InfluxDB `org`.